pkgs/development/python-modules/mkdocs-include-markdown-plugin/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hatchling
+, mkdocs
+, wcmatch
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-include-markdown-plugin";
+  version = "6.0.4";
+
+  src = fetchFromGitHub {
+    owner = "mondeja";
+    repo = "mkdocs-include-markdown-plugin";
+    rev = "v${version}";
+    sha256 = "sha256-wHaDvF+QsEa3G5+q1ZUQQpVmwy+oRsSEq2qeJIJjFeY=";
+  };
+
+  pyproject = true;
+  doCheck = false;
```

Comment:
Specify on a comment above why the checks are disabled. The repo contains tests.